data/json/enchantments.json

```diff
@@ -194,11 +194,11 @@
     "id": "combat_exoskeleton_heavy_active",
     "type": "enchantment",
     "name": { "str": "Active combat exoskeleton mk.1" },
-    "description": "The exoskeleton helps you carry more weight, protects you against harmful gas, cools you down, and has reduced encumbrance.",
+    "description": "Now powered, the exoskeleton provides <color_white>significant</color> improvements to your carry weight and movement exertion while the life support system shields you against harmful gas, cools you down, and dampens harmful sounds.",
```

Comment:
```suggestion
    "description": "Now powered, the exoskeleton provides <color_white>significant</color> improvements to your carry weight and movement exertion while the life support system shields you against harmful gas, cools you down and dampens harmful sounds.",
```

Reply:
That's an oxford comma. The sentence is correct either way - I honestly prefer having it for the sake of pacing since the clause before it is so long.

Reply:
Alright, feel free to dismiss the review then.